src/EprPrnIntegration.Api/FetchNpwdIssuedPrnsFunction.cs

```diff
@@ -35,8 +58,14 @@ public async Task Run([TimerTrigger("%FetchNpwdIssuedPrns:Schedule%")] TimerInfo
 
             _logger.LogInformation($"FetchNpwdIssuedPrnsFunction function started at: {DateTime.UtcNow}");
 
-            //pending find out the filter criteria and add it
-            var filter = "1 eq 1";
+            var deltaRun = await _utilities.GetDeltaSyncExecution(NpwdDeltaSyncType.FetchNpwdIssuedPrns);
+            var toDate = DateTime.UtcNow;
+            var filter = "(EvidenceStatusCode eq 'EV-CANCEL' or EvidenceStatusCode eq 'EV-AWACCEP' or EvidenceStatusCode eq 'EV-AWACCEP-EPR')";
+
+            if (deltaRun != null && deltaRun.LastSyncDateTime > DateTime.Parse(_configuration["DefaultLastRunDate"]!))
```

Comment:
since we are in if condition, its better to useTryParse to make sure we are covered by the non-datatime value in the config value as well